config/default.yml

```diff
@@ -283,6 +283,24 @@ PreCommit:
     description: 'Checking YAML syntax'
     include: '**/*.yml'
 
+  W3cCssValidator:
+    enabled: false
+    description: 'Analyzing with W3C CSS validation service'
+    validator_uri: 'http://jigsaw.w3.org/css-validator/validator'
+    language: 'en'
+    profile: 'css3'
+    warn_level: 2
+    include:
+      - '**/*.css'
+
+  W3cHtmlValidator:
```

Comment:
This should also be opt-in (`enabled: false`). It's pretty heavyweight since it makes a remote request, and some organizations would have policies on sending private code to external entities.